src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -454,7 +454,7 @@ public function setStatusCode($code, $text = null)
         }
 
         if (null === $text) {
-            $this->statusText = isset(self::$statusTexts[$code]) ? self::$statusTexts[$code] : '';
+            $this->statusText = isset(self::$statusTexts[$code]) ? self::$statusTexts[$code] : 'Unknown status text';
```

Comment:
or maybe all lowercase to differenciate from regular status texts that are "ucfirst"